javascript/javascript3/week1/Giphy.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>giphy</title>
+</head>
+<body>
+    <h1>Giphy Finder</h1>
+    <form id="searchForm">
```

Comment:
I tried to run your code by saving it and running it in browser. And I faced one problem - it didn`t work until I deleted this line with code `form id="searchForm"`. So my suggestion is to change it to a div element that contains the rest of the info.